src/firefly/js/visualize/ui/CatalogSelectViewPanel.jsx

```diff
@@ -28,6 +28,7 @@ import {FileUpload} from '../../ui/FileUpload.jsx';
 import {convertAngle} from '../VisUtil.js';
 
 import './CatalogTableListField.css';
+import './CatalogSelectViewPanel.css';
```

Comment:
i know it's recommended to have the css file close to the component.  but in this case, i think it's better to combine them with their top-level component(CatalogSelectViewPanel.css)